src/types.jl

```diff
@@ -12,18 +11,48 @@ struct SCSMatrix
     n::Int
 end
 
+# Version where Julia manages the memory for the vectors.
+struct ManagedSCSMatrix
+    values::Vector{Cdouble}
+    rowval::Vector{Int}
+    colptr::Vector{Int}
+    m::Int
+    n::Int
+end
+
+function ManagedSCSMatrix(m::Int, n::Int, A::SCSVecOrMatOrSparse)
+    A_sparse = sparse(A)
+
+    values = copy(A_sparse.nzval)
+    rowval = convert(Array{Int, 1}, A_sparse.rowval .- 1)
+    colptr = convert(Array{Int, 1}, A_sparse.colptr .- 1)
+
+    return ManagedSCSMatrix(values, rowval, colptr, m, n)
+end
+
+# Returns an SCSMatrix. The vectors are *not* GC tracked in the struct.
+# Use this only when you know that the managed matrix will outlive the SCSMatrix.
+SCSMatrix(m::ManagedSCSMatrix) =
+    SCSMatrix(pointer(m.values), pointer(m.rowval), pointer(m.colptr), m.m, m.n)
+
 
 struct SCSSettings
     normalize::Int # boolean, heuristic data rescaling: 1
     scale::Cdouble # if normalized, rescales by this factor: 1
     rho_x::Cdouble # x equality constraint scaling: 1e-3
-    max_iters::Int # maximum iterations to take: 2500
-    eps::Cdouble # convergence tolerance: 1e-3
+    max_iters::Int # maximum iterations to take: 5000
+    eps::Cdouble # convergence tolerance: 1e-5
```

Comment:
Is this related to #59  ? The value does not match what I say in the issue. Has it changed with the latest version of SCS ? The README should probably also be updated

Reply:
The defaults have changed, but it's not a good idea to have them listed in three places. I removed them from the README.